src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -288,9 +288,81 @@ boolean canAddField() {
       }
     }
   }
+
+  /**
+   * Filters db documents based on non document id field values if id use "$in" operator, use
+   * IDFilter, since partition key 'or' is not supported besides id, "$in" operator, use INFilter
+   */
+  public static class InFilter extends DBFilterBase {
+    private final Object arrayValue;
```

Comment:
Can we not use anything more specific than `Object`

Reply:
solved, use List<Object>